src/State/Hydration/reducers.ts

```diff
@@ -4,8 +4,9 @@ export const initialState = {
   isDestinationModalOpen: false,
   isEditorOpen: false,
   selectedNode: null,
-  metaData: {
-    sourceTiles: {}
+  metadata: {
+    sourceTiles: {},
```

Comment:
is there a reason for the naming of "sourceTiles", why not sources? just wondering

Reply:
changed to `sources`